src/mmw/apps/modeling/migrations/0020_remove_project_is_activity.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('modeling', '0019_project_gis_data'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Trying to determine the implications of this schema change. In a scenario where the old release attempts to interact with the database that is in a state after this migration is applied, it'll fail when a new project is created?